recipes/agent.rb

```diff
@@ -122,9 +122,10 @@
 end
 
 # Client utilities for pulp content manager
-package "python3-pip"
-%w[pulp-rpm-client==3.9.0 pulpcore-client==3.9.1].each do |pulppkg|
-  execute "python3 -m pip install #{pulppkg}"
+%w[pulp-rpm-client pulpcore-client].each do |pulppkg|
```

Comment:
Maybe worth marking this to be removed in the next release.

Reply:
Done in 282425188618b1619de5d85871985f072e76abf8